tests/test_properties.py

```diff
@@ -113,6 +113,13 @@ async def test_roundtrip_array_metadata(
         assert actual == expected
 
 
+@given(store=stores, meta=array_metadata())  # type: ignore[misc]
+def test_array_metadata_meets_spec(store: Store, meta: ArrayV2Metadata | ArrayV3Metadata) -> None:
```

Comment:
since this test does nothing for v3 metadata, would it be possible to only run it on `ArrayV2Metadata`?

Reply:
here I made it do something ;) . really we should fill it out.

Reply:
i was more thinking that it would make sense to test the v2 and v3 specs in separate tests, since they are logically independent, but we can do that refactor later